### ISSUE TITLE
Missing `.pbxproj` file?

### ISSUE THREAD

**Issue by andrejbauer** [markokoleznik/agda-writer#2]
When I try to open `AgdaWriter.xcodeproj` Xcode says:

```
Project /Users/andrej/Documents/agda-writer/AgdaWriter.xcodeproj
cannot be opened because it is missing its project.pbxproj file.
```

Now what?

**Comment by markokoleznik** [comment]
My bad :) 
Added too much files to .gitignore 

Does it work now?

**Comment by andrejbauer** [comment]
Closed by 7c2ae7dbeea4f56185044eec543e35491f0222ba.
